packages/gtest.rb

```diff
@@ -0,0 +1,52 @@
+require 'package'
+
+class Gtest < Package
+  description 'Google Test - C++ testing utility'
+  version '1.10.0-389c'
+  compatibility 'all'
+  source_url 'https://github.com/google/googletest/archive/389cb68b87193358358ae87cc56d257fd0d80189.zip'
+  source_sha256 '763e20249e76417bed7ebc44aa85fedf5fbac6f9fb6d30bddb628ab07ebf04f5'
+
+  binary_url ({
+     aarch64: 'file:///usr/local/tmp/packages/gtest-1.10.0-389c-chromeos-armv7l.tar.xz',
+      armv7l: 'file:///usr/local/tmp/packages/gtest-1.10.0-389c-chromeos-armv7l.tar.xz',
+        i686: 'file:///usr/local/tmp/packages/gtest-1.10.0-389c-chromeos-i686.tar.xz',
+      x86_64: 'file:///usr/local/tmp/packages/gtest-1.10.0-389c-chromeos-x86_64.tar.xz',
```

Comment:
Not Bintray binaries?

Reply:
Whoops

Reply:
Fixed. This is what happens with multiple remote edit windows open.